src/services/api/activity-log/activity.log.module.ts

```diff
@@ -45,6 +45,6 @@ import { CalendarEventModule } from '@domain/timeline/event/event.module';
     ActivityLogResolverQueries,
     ActivityLogResolverSubscriptions,
   ],
-  exports: [],
+  exports: [ActivityLogService], // todo
```

Comment:
todo?

Reply:
I introduced it because this service was not meant to be used outisde of the module.